config/permission.php

```diff
@@ -77,4 +77,25 @@
      */
 
     'cache_expiration_time' => 60 * 24,
+
+
+    /*
+     * By default we'll make an entry in the application log when the permissions
+     * could not be loaded. Normally this only occurs while installing the packages.
+     *
+     * If for some reason you want to disable that logging, set this value to false.
+     */
+
+    'log_registration_exception' => true,
```

Comment:
I think you just merged in something that was removed in the release published earlier today ... :(

Reply:
hmmm, so we need to remove `log_registration_exception`?